services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/service/impl/NeighborService.java

```diff
@@ -113,8 +117,9 @@ public Neighbor.NeighborState buildNeighborState(NeighborEntry.NeighborType type
         fixedIpBuilder.setIpAddress(portHostInfo.getPortIp());
         fixedIpBuilder.setNeighborType(neighborType);
         if (arionGatwayEnabled) {
+            neighborConfigBuilder.setHostMacAddress(nodeInfoCache.getNodeInfoByNodeIp(portHostInfo.getHostIp()).get(0).getMacAddress());
```

Comment:
Is it possible that nodeInfoCache.getNodeInfoByNodeIp(portHostInfo.getHostIp()) is empty?

Reply:
Fixed